var/spack/repos/builtin/packages/py-pyyaml/package.py

```diff
@@ -13,8 +13,16 @@ class PyPyyaml(PythonPackage):
     pypi = "PyYAML/PyYAML-5.3.1.tar.gz"
```

Comment:
If you change this to lowercase, `spack versions` and `spack checksum` will be able to find future versions.

Reply:
Ahh. So we "lose" the ability to see the older mixed-case ones in `spack checksum` but future checksumming is nicer. Okay I'll test that out.

Reply:
Change pushed. Smart idea!